recipes/management/increasing_the_capacity_to_win/growing/expectations/onboarding/README.md

```diff
@@ -0,0 +1,34 @@
+# Onboarding
+When employees get hired or change roles within a company the process of ramping them up is called 'onboarding'. Sometimes this is done by the 'deep end of the pool' approach (just throw them in and see if they can swim). Sometimes it's done methodically, carefully. Sometimes its done too conservatively. Read through the recipes here and see what flavor feels like you...
```

Comment:
Hm... this is framed very  "onboarding manager" centric. I wonder if towards the end you may encourage a person to consider who they're onboarding and which approach may be the most useful?